dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/impl/ShowClaimItemFeature.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.authorization.impl;
+
+import java.sql.SQLException;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.dspace.app.profile.service.ResearcherProfileService;
+import org.dspace.app.rest.authorization.AuthorizationFeature;
+import org.dspace.app.rest.authorization.AuthorizationFeatureDocumentation;
+import org.dspace.app.rest.model.BaseObjectRest;
+import org.dspace.app.rest.model.ItemRest;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Item;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Checks if the given user can request the claim of an item. Whether or not the
+ * user can then make the claim is determined by the feature
+ * {@link CanClaimItemFeature}.
+ *
+ * @author Corrado Lombardi (corrado.lombardi at 4science.it)
+ * @author Luca Giamminonni (luca.giamminonni at 4science.it)
+ */
+@Component
+@AuthorizationFeatureDocumentation(name = ShowClaimItemFeature.NAME,
+    description = "Used to verify if the given user can request the claim of an item")
+public class ShowClaimItemFeature implements AuthorizationFeature {
```

Comment:
The showClaimItemFeature is used to show the button for claim an item. The canClaimItemFeature then it is used to verify if the claim can actually be made. This distinction allows first of all to give more feedback to the user if for example he wants to try to make the claim of a profile but this is already associated with another user. Furthermore, it could also allow in the future to adopt more complex logic on the possibility of making a claim.

Reply:
@LucaGiamminonni : Please add these details/differences to the JavaDocs for these Features then.  Currently, both `CanClaimItemFeature` and `ShowClaimItemFeature` seem extremely similar...and the differences are unclear.  This is the first "Feature" that I'm aware of that has both a `Can*` and a `Show*`.  In other features, usually the UI just checks `Can*` to also determine whether to show the button.  This scenario seems unique, so it needs to be described better.